test/extended/clusterup.sh

```diff
@@ -0,0 +1,215 @@
+#!/bin/bash
+source "$(dirname "${BASH_SOURCE}")/../../hack/lib/init.sh"
+trap os::test::junit::reconcile_output EXIT
```

Comment:
i have not had time to factor this out yet, so you will need to cargo-cult the following for now:

https://github.com/openshift/origin/blob/master/test/extended/gssapi.sh#L21-L24
https://github.com/openshift/origin/blob/master/test/extended/gssapi.sh#L38-L55